yktrace/src/errors.rs

```diff
@@ -3,11 +3,13 @@ use std::fmt::{self, Display, Formatter};
 #[derive(Debug)]
 /// Reasons that a trace can be invalidated.
 pub enum InvalidTraceError {
+    /// The trace came back empty.
+    EmptyTrace,
+    /// Something went wrong in the compiler's tracing code
+    InternalError,
     /// There is no SIR for the location in the trace.
     /// The string inside is the binary symbol name in which the location appears.
-    NoSir(String),
-    /// Something went wrong in the compiler's tracing code
-    InternalError
+    NoSir(String)
```

Comment:
```suggestion
    NoSir(String),
```

Reply:
Funny that `rustfmt` doesn't canonicalise that.

Reply:
Ah wait, it does. It removes the comma.

Reply:
That is because rustfmt is configured to remove it. The default configuration adds it.

https://github.com/softdevteam/yk/blob/8d57b6239062ff17cd5eeb0cea70516d47948ddf/yktrace/.rustfmt.toml#L3

Reply:
Ah I thought we'd got rid of the custom `.rustfmt.toml`s but maybe not!

Reply:
Oh! Let's kill it.

Reply:
Yes, but that should be a separate PR.

Reply:
Ah. I already made the commit with the intention of keeping it separate. Shall I revert?